### PR TITLE
Disallow the next button when the intervention has not been interacted with

### PR DESCRIPTION
**Pull request by jrosen081**
This fixes the issue that @stefan-philip reported.

### CODE REVIEW

File: frontend/src/pages/Interventions/FirstActivity.tsx [hunk 1]
```diff
@@ -100,6 +100,7 @@ const FirstActivity = ({
                 top={20}
                 delay={2000}
                 onClick={updateIntervention}
+                canBeShown
```
Comment (cindy1u0) [inline frontend/src/pages/Interventions/FirstActivity.tsx:103]: ig technically we don't really need canBeShown for first activity and fourth activity but I think it's okay to just leave it

**Review by cindy1u0 — APPROVED** [review]
LGTM ~

**Review by stefan-philip — CHANGES_REQUESTED** [review]
On the yes-or-no activity, when you select yes or no the prompt is played again

**Comment by jrosen081** [comment]
@stefan-philip Good catch. This should be fixed now

**Review by stefan-philip — APPROVED** [review]
Awesome!

**Pull request merged**